collectors/bzimport/convertors.py

```diff
@@ -784,12 +817,35 @@ def get_affects(self, flaw):
             self.record_errors(errors)
             affects.append(affect_obj)
 
+            # AffectCVSS is created here because is requires an affect object
+            for cvss_pair in [
+                ("cvss2", AffectCVSS.CVSSVersion.VERSION2),
+                ("cvss3", AffectCVSS.CVSSVersion.VERSION3),
+            ]:
+                cvss = cvss_pair[0]
+                version = cvss_pair[1]
+
+                if affect_json.get(cvss) and "/" in affect_json[cvss]:
+                    cvss_obj = AffectCVSS.objects.create_cvss(
```

Comment:
I expect that affects do not have comments for cvss scores.

Reply:
This is a correct assumption in the Bugzilla world.